### PR TITLE
Akb93 week8

### PR DESCRIPTION
**Pull request by AdamKBeck**

### CODE REVIEW

File: postr/instagram_postr.py [hunk 1]
```diff
@@ -25,15 +32,31 @@ def __init__(self, user: Dict[str, Any]) -> None:
 class Instagram(ApiInterface):
     """ Wrapper for accessing the instagram API  """
 
+    # A workaround URL that retrieves a profile JSON without triggering a 403 error.
+    # URL before the username
+    PRE_PROFILE_URL = 'https://www.instagram.com/web/search/topsearch/?context=blended&query='
+    # URL after the username, which loads the profile JSON
+    POST_PROFILE_URL = '&rank_token=0.3953592318270893&count=1'
```
Comment (dbgrigsby) [inline postr/instagram_postr.py]: move the 0.39.......93 to a config file?

File: postr/instagram_postr.py [hunk 2]
```diff
@@ -72,6 +95,16 @@ def refresh(self) -> None:
         self.followers = self._user_follower_info()
         self.followings = self._user_following_info()
 
+    @staticmethod
+    def direct_share(mediaID: str, recipients: List[int], message: str = '') -> None:
```
Comment (dbgrigsby) [inline postr/instagram_postr.py]: mediaID -> media_id

File: postr/instagram_postr.py [hunk 3]
```diff
@@ -72,6 +95,16 @@ def refresh(self) -> None:
...
+        """
+        Shares media to a list of recipients via a direct message
+        mediaID: The id of the media to share
+        recipients: A list of the user ids to share media with
+        mesage: The message to go along with the media share
+        """
+        InstagramAPI.direct_share(mediaID, recipients, message)
+
     def spam_follower_ratio(self, uid: int = 0) -> float:
```
Comment (dbgrigsby) [inline postr/instagram_postr.py]: why is uid not used?

File: postr/instagram_postr.py (hunk 1 above)
Comment (dbgrigsby) [reply postr/instagram_postr.py]: or extract it to its own constant with a description of the problem / link to issue where found

File: postr/instagram_postr.py (hunk 3 above)
Comment (AdamKBeck) [reply postr/instagram_postr.py]: direct_share takes in a list of usernames, no uid is used. The API is inconsistent.

**Pull request merged**